### PR TITLE
Build backend

### PR DESCRIPTION
**Pull request by amon-cofie**
In this PR I did the following things: 

- Created a ruby on rails API only project
- Added a model for the greetings table
- Added the API controller in the api/v1 namespace
- Added the rack-cors gem to enable resource sharing with the react frontend

This is a link to the [frontend pull request](https://github.com/amon-cofie/react-rails-frontend/pull/1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,183 @@
...
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
```
Comment (Deepakdanger) [inline README.md:118]: - [ ] I would recommend mentioning the port where one can run the rails application. It should run on the 8000 port to run perfectly.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,183 @@
...
 
-* Deployment instructions
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>REACT</li>
+  </ul>
+</details>
 
-* ...
```
Comment (Deepakdanger) [inline README.md:24]: - [ ] I recommend mentioning the frontend app repo in the README as well, As this is where all people look to find the frontend application. Also, mention the backend repo in the frontend README.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @amon-cofie, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @amon-cofie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**